<!-- image -->
 doi: 10.1093/toxsci/kfx169 Advance Access Publication Date: August 16, 2017 Research Article
 <!-- image -->
 ## Developmental Neurotoxicants Disrupt Activity in Cortical Networks on Microelectrode Arrays: Results of Screening 86 Compounds During Neural Network Formation
 Christopher L. Frank, Jasmine P. Brown, Kathleen Wallace, William R. Mundy, and Timothy J. Shafer 1
 Integrated Systems Toxicology Division, National Health and Environmental Effects Research Laboratory, US EPA, MD105-05, Element Genomics, Inc, Durham, NC, Research Triangle Park, North Carolina 27711
 1 To whom correspondence should be addressed. Fax: (919) 541-4849. E-mail: shafer.tim@epa.gov.
 ## ABSTRACT
 Less than 1% of environmental chemicals have been evaluated for developmental neurotoxicity (DNT). Current guideline DNT studies are resource intensive and not amenable to screening large numbers of compounds for hazard. As part of evaluating a battery of more rapid and scalable in vitro assays for DNT hazard, 86 compounds were screened for their ability to alter function during cortical network development. Developing rat cortical networks were treated with a concentration series (usually 0.03-30 m M) of 86 compounds, 60 of which have known in vivo DNT effects ('DNT Reference Set'). Spontaneous network activity was monitored by microelectrode array recordings over 12days in vitro , and 17 measures of network activity and synchrony were quantified. Following recordings on days in vitro 12, in-well cell assessment of metabolic activity (Alamar blue) and total cellular content (lactase dehydrogenase) were conducted. Of the 86 compounds tested, 64 perturbed cortical network function in a concentration-dependent manner; 49 of the 60 DNT Reference Set compounds (81.7%) altered network formation. Compounds were ranked by potency (network effect EC50) and selectivity (separation of network and cell viability EC50) for hazard prioritization. Machine learning indicates a combination of an overall network activity metric with a measure of network coordination is key in distinguishing network-disruptive from benign treatments. These data demonstrate that this microelectrode array-based assay for developing cortical network function is amenable to medium-throughput evaluation of environmental substances for DNT hazard and further prioritization. For comprehensive identification of compounds of concern, this assay will be a useful component of a battery of assays targeting independent neurodevelopmental processes.
 Key words: screening; microelectrode array; cell culture; in vitro and alternatives; neurotoxicity; developmental; neurotoxicology.
 There is concern that exposure to environmental chemicals may contribute to a number of neurodevelopmental disorders in children (Braun et al. , 2006; Grandjean and Landrigan, 2006, 2014; Hertz-Picciotto et al. , 2006; Karr, 2012; Pola /C19 nska et al. , 2012). However, evaluation of developmental neurotoxicity (DNT) hazard has progressed slowly in comparison to other adverse health outcomes. This is primarily due to a lack of reliable, rapid, and cost-efficient screening methods for neurodevelopmental endpoints, as well as the complexity of developmental processes that occur in a spatially and temporally coordinated manner (Judson et al. , 2009; Makris et al. , 2009). Current regulatory guidelines for DNT testing require costly studies involving hundreds of laboratory animals and a timeline of many months to years for each compound evaluated, severely limiting
 |
 throughput of these studies (Smirnova et al. , 2014). Recognizing the need for improved screening, several groups have developed various molecular, cellular, and alternative species-based assays focused upon key neurodevelopmental processes (Harrill et al. , 2010, 2011; Jarema et al. , 2015; Johnstone et al. , 2010; Krug et al. , 2013; Schmuck et al. , 2017; Stiegler et al. , 2011; Zimmer et al. , 2014). By testing environmental compounds in a panel of more efficient in vitro assays, each technically validated for a particular neurodevelopmental process, an aggregate DNT hazard score could be assigned to prioritize compounds more rapidly for further investigation.
 The culmination of neurodevelopment is the formation of functional neural networks. Connectivity and coordination of neural networks are key to fundamental cognitive processes such as attention, learning, and memory (Buschman and Kastner, 2015; Korte and Schmitz, 2016; Salinas and Sejnowski, 2001). An assay that directly measures successful formation of neural networks may therefore be advantageous to identify potential developmental neurotoxicants. To this end, we recently developed an assay for network development based on microelectrode array (MEA) recordings of primary rat cortical neurons over 12days in vitro (DIV) (Brown et al. , 2016). We previously demonstrated proof of concept for this approach by testing 5 compounds chosen for their ability to disrupt normal cortical development and/or network formation and 1 negative control compound. This study demonstrated the assay's ability to discriminate between positive and negative control compounds in at least 16 measures of network activity, identify concentrations at which network failure begins when testing in a concentration-response scheme, and identify selective network perturbations versus nonspecific cytotoxic effects when coupled with terminal cell death assays. Together, these features facilitate chemical prioritization on the basis of potency and selectivity of network developmental effects.
 The goal of the current study was 2-fold: first, to demonstrate that the throughput of this assay is amenable to screening compounds and second, to evaluate the ability of the assay to detect known neurodevelopmental toxicants on the basis of neural network disruption. Thus, we used this network formation assay to test 86 unique compounds, representing a broad range of DNT mechanisms and possible molecular initiating events. Sixty of the 86 compounds selected have in vivo evidence of DNT effects, ranging from behavioral effects to gross morphological defects to more nuanced neurochemical imbalances (Mundy et al. , 2015) and thus comprise a 'DNT Reference Set' of compounds. Together, these 60 compounds reflect the diversity of known DNT hazards that a battery of in vitro assays would ideally identify. We hypothesize that these compounds will disrupt activity during network formation at a higher rate than non-neurotoxic compounds. Another 21 compounds were selected as a panel of unknowns that lack sufficient literature evidence of DNT effects but included 5 compounds that are expected negatives because they are generally considered safe (ie, approved for use during pregnancy or as common food additives). Finally, 4 positive control compounds and 1 negative control, all tested previously in the assay (Brown et al. , 2016), were included to confirm assay reproducibility. Primary rat cortical neurons were cultured in the presence of the compounds as they developed functional networks over 12 DIV. Multiple MEA recordings of each developing network were made over this time, and effects of these compounds on 17 parameters of network development and 2 measures of cell viability were determined.
 ## MATERIALS AND METHODS
 Compounds. The present study evaluated 86 compounds (Table 1); 60 of which have literature evidence of DNT effects in humans or animal models (Mundy et al. , 2015). In addition, 4 compounds tested previously in this assay (bisindolylmaleimide I, loperamide, mevastatin, and sodium orthovandate) were used as assay positive controls with some network effects expected (Brown et al. , 2016). Acetaminophen was included as an assay negative control. The remaining 21 compounds lack evidence of DNT effects in the published literature and therefore represent unknowns. At least 4 of the unknown compounds (1,2-propylene glycol, amoxicillin, sodium benzoate, and sorbitol) are generally considered safe (ie, propylene glycol, sodium benzoate, and sorbitol are listed as Food and Drug Administration [FDA] Generally Regarded as Safe food substances; amoxicillin is FDA pregnancy risk Category B) (Briggs et al. , 2008; US FDA, 2015). The remaining unknown compounds were selected because they were included in the EPA Toxcast project and/or have previously been tested for acute toxicity on MEAs (Behl et al. , 2015; Sipes et al. , 2013; Valdivia et al. , 2014). Compounds were stored as specified by the manufacturer (Table 1) and dissolved in dimethylsulfoxide (DMSO), water, or ethanol (Table 1) to a stock concentration of 30 mM and serially diluted by half-log increments. Purity of each compound was 95% or greater. Most compounds were tested at concentrations ranging from 0.03 to 30 m M, with some compounds tested at lower concentrations when it was found that the standard concentration range was overtly cytotoxic (Table 1).
 Cell culture and plate design. All procedures involving animals were approved by the National Health and Environmental Effects Research Laboratory Institutional Animal Use and Care Committee. Primary cultures were prepared from the cortex of 0 to 24-hour-old postnatal Long Evans rat pups. The culture has been described previously and is comprised of excitatory and inhibitory neurons and glia (Bjo ¨rklund et al. , 2010; Mundy and Freudenrich, 2000). Harvesting and seeding of the cells is described by Valdivia et al. (2014). Cortical cells were plated at a density of 150000 cells in a 25 m l Neurobasal A plus laminin (20 m g/ml; Sigma-Aldrich, Cat L2020)drop placed directly over the array field in each well of a 48-well MEA plate (Axion M768KAP-48) precoated with 0.05% polyethylenimine. Cultures were maintained in Neurobasal A media (Gibco, Cat 10888022) supplemented with B-27 (Gibco, Cat 17504044).
 Six compounds were tested per 48-well MEA plate with 7 concentrations used per compound (see Table 1 for exact concentration range used). As outlined for screening approaches (Malo et al. , 2006), each compound was tested a minimum of 3 times on 3 separate MEA plates from the same culture and each well was considered 1 replicate for statistical analyses. It has been shown previously with 6 compounds that independent cultures produce highly similar results in this assay (Brown et al. , 2016); further, 7 other compounds tested here were tested more than once with similar results (replicate data not shown; data presented are the averages of all determinations). An adjacent untreated control well was included for each row on the plate for 6 total untreated wells per plate. All statistical analyses first normalized well level values to same plate controls. Stock solutions were prepared at 1000 /C2 the final concentration range in sterile solvent and diluted first to 50 /C2 in a 96-well dosing plate (5 m l compound and 95 m l sterile Neurobasal A media) and then 1:50 into 500 m l media placed into the appropriate well of the 48-well MEA plate 2hours after cell attachment.
 Table 1. Compounds Tested in This Study
 |
 | Storage ( /C14 C)                 | 25 /C0 20 25 25 /C0 20                                                                   | 4 25 25 25 25 25 4 25 25 25 25 25 4 25 25 25 25 25 25 25 4 25 4 25 4 25 25 25 25 25 25 4 25 25 25 25 25                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | |-----------------------------------|------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| | Source                            | Sigma-Aldrich Calbiochem Sigma-Aldrich EMD Millipore Sigma-Aldrich                       | Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Thermo Fisher Scientific Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Calbiochem Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Battelle Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich | | Solvent used                      | DMSO DMSO DMSO DMSO Water                                                                | DMSO DMSO DMSO DMSO DMSO DMSO Water DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO Water DMSO DMSO DMSO DMSO Water DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO DMSO                                                                                                                                                                                                                                                                                                                                                                                           | | Rationale for selection           | Assay negative b Assay positive b Assay positive b Assay positive b Assay                | positive b DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a                                                                                                                                                                                                                                                                                                         | | Concentration range tested ( m M) | 0.03-30 0.01-10 0.03-30 0.03-30 0.03-30                                                  | 0.01-10 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.001-1 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30                                                                                                                                                                                                                                                                               | | DTXSID                            | DTXSID2020006 DTXSID50157932 DTXSID6045165 DTXSID4040684 DTXSID2037269                   | DTXSID3030056 DTXSID2020634 DTXSID0034300 DTXSID5020027 DTXSID0039223 DTXSID30157298 DTXSID4022600 DTXSID5020607 DTXSID9020166 DTXSID7020182 DTXSID1023940 DTXSID0020232 DTXSID4022731 DTXSID9020247 DTXSID4046022 DTXSID0022808 DTXSID4020458 DTXSID1038666 DTXSID2038443 DTXSID5024845 DTXSID5020364 DTXSID1023998 DTXSID3022877 DTXSID8020381 DTXSID3020384 DTXSID4020406 DTXSID9020453 DTXSID3020465 DTXSID8020541 DTXSID0034566 DTXSID7023067 DTXSID4034150 DTXSID1024126 DTXSID6020690 DTXSID6025438 DTXSID5032442 DTXSID8023187                                                | | CAS No.                           | 103-90-2 133052-90-1 53179-11-6 73573-88-3 13721-39-6                                    | 5436-43-1 51-21-8 135410-20-7 79-06-1 116-06-3 329-89-5 51-63-8 117-81-7 56-35-9 80-05-7 7790-78-5 58-08-2 298-46-4 63-25-2 438-41-5 69-09-0 2921-88-2 5598-15-2 50-36-2 64-86-8 6055-19-2 52315-07-8 147-94-4 52918-63-5 50-02-2 439-14-5 60-57-1 56-53-1 57-41-0 155569-91-8 56296-78-7 52-86-8 1024-57-3 70-30-4 127-07-1 138261-41-3 33795-24-3                                                                                                                                                                                                                                   | | Compound name (abbreviation)      | Acetaminophen Bis -indolylmaleimide I (Bis 1) Loperamide Mevastatin Sodium orthovanadate | 2,2'4,4'-Tetrabromodiphenyl ether 5-Fluorouracil Acetamiprid Acrylamide Aldicarb Aminonicotinamide Amphetamine Bis (2-ethylhexyl) phthalate (DEHP) Bis (tri-n-butylin) oxide Bisphenol A Cadmium Caffeine Carbamazepine Carbaryl Chlordiazepoxide Chlorpromazine Chlorpyrifos Chlorpyrifos oxon Cocaine Colchicine Cyclophosphamide Cypermethrin Cytosine arabinoside Deltamethrin Dexamethasone Diazepam Dieldrin Diethylstilbestrol Diphenylhydantoin Emamectin benzoate Fluoxetine Haloperidol Heptachlor epoxide Hexachlorophene Hydroxyurea Imidacloprid Ketamine                |
 ## Table 1. (continued)
 |
 | Storage ( /C14 C)            | /C0 20 25 25 25 /C0 20 25 4 /C0 20 25 25 25 25 25 25 4 25 25 /C0 20 25 25 25 25 20 25 25 25 4 25 25 25 25 4 25 25 4 4 25 4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | |------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| | Source                       | Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Thermo Fisher Scientific Sigma-Aldrich Honeywell Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich MP Biomedicals MRIGlobal Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich Chem Service, Inc. Sigma-Aldrich Chem Service, Inc. Chem Service, Inc. Sigma-Aldrich Chem Service, Inc. Santa Cruz Biotech Sigma-Aldrich Sigma-Aldrich Sigma-Aldrich                                                                                                                                                                                                                                                                                                                                                  | | Solvent used                 | DMSO Ethanol DMSO Water DMSO DMSO Water DMSO Water DMSO DMSO Water Water DMSO DMSO DMSO DMSO DMSO DMSO Water DMSO DMSO Water DMSO DMSO DMSO DMSO DMSO DMSO/ethanol DMSO DMSO Water DMSO DMSO Water Water DMSO Water                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | | m M) Rationale for selection | DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a DNTE a , assay positive Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | | Concentration range tested ( | 0.03-30 0.03-30 0.01-10 0.03-30 0.03-30 0.01-10 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.01-10 0.01-10 0.03-30 0.03-30, 3-3000 0.01-10 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30 0.03-30                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | | No. DTXSID                   | 6080-56-4 DTXSID10274073 58-89-9 DTXSID2020686 12427-38-2 DTXSID9020794 7773-01-5 DTXSID2024169 59-05-2 DTXSID4020822 22967-92-6 DTXSID9024198 51481-60-8 DTXSID8023349 54-11-5 DTXSID1020930 4685-14-7 DTXSID3034799 52645-53-1 DTXSID8022292 57-30-7 DTXSID5021122 7784-46-5 DTXSID5020102 7681-49-4 DTXSID2020630 107534-96-3 DTXSID9032113 23031-32-5 DTXSID7021310 50-35-1 DTXSID9022524 51-52-5 DTXSID4021347 302-79-4 DTXSID7021239 2767-54-6 DTXSID9040712 56-24-6 DTXSID6035102 115-86-6 DTXSID1021952 1069-66-5 DTXSID6023733 14277-97-5 DTXSID20274180 26787-78-0 DTXSID3037044 62571-86-2 DTXSID1037197 133-90-4 DTXSID2020262 486-56-6 DTXSID1047576 111-46-6 DTXSID8020462 120068-37-3 DTXSID4034609 86386-73-4 DTXSID3020627 85509-19-9 DTXSID3024235 1071-83-6 DTXSID1024122 54-85-3 DTXSID8020755 77501-63-4 DTXSID7024160 26172-55-4 DTXSID9034286 108-95-2 DTXSID5021124 82385-42-0 DTXSID5021251 532-32-1 DTXSID1020140 | | Compound name (abbreviation) | Lead acetate Lindane Maneb Manganese Methotrexate Methylmercury Naloxone Nicotine Paraquat Permethrin Phenobarbital Sodium arsenite Sodium Fluoride Tebuconazole Terbutaline Thalidomide Thiouracil t-retinoic acid Triethyltin bromide Trimethyltin Triphenyl phosphate (TPHP) Valproic acid Domoic acid Amoxicillin Captopril Chloramben Cotinine Diethylene glycol Fipronil Fluconazole Flusilazole Glyphosate Isoniazid Lactofen Methylchloroisothiazolinone Phenol Saccharin Sodium benzoate                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 Table 1. (continued)
 | Storage ( /C14 C)   |                    | 25          | 25                  | 25                      | 25               | 25            | |---------------------|--------------------|-------------|---------------------|-------------------------|------------------|---------------| | Source              | Chem Service, Inc. |             | Sigma-Aldrich       | Sigma-Aldrich           | MRIGlobal        | Sigma-Aldrich | | Solvent used        |                    | DMSO        | DMSO                | DMSO                    | DMSO             | DMSO          | | Rationale for       | Unknown            |             | Unknown             | Unknown                 | Unknown          | Unknown       | | Concentration       |                    | 0.03-30     | 0.03-30             | 0.03-30                 | 0.03-30          | 0.03-30       | | DTXSID              | DTXSID1034212      |             | DTXSID1026081       | DTXSID9026261           | DTXSID5021411    | DTXSID0021206 | | CAS No.             | 118134-30-8        |             | 79-94-7             | 13674-87-8              |                  |               | | (abbreviation)      |                    |             |                     | phosphate (TDCIPP)      | 115-96-8         | 57-55-6       | |                     |                    |             | A (TBBPA)           |                         | phosphate (TCEP) |               | | name                |                    |             |                     | (1,3-dichloro-2-propyl) |                  | glycol        | |                     |                    |             |                     |                         | (2-chloroethyl)  |               | |                     |                    | Spiroxamine | Tetrabromobisphenol |                         |                  | 1,2-Propylene | | Compound            |                    |             |                     |                         |                  |               | |                     |                    |             |                     | Tris                    |                  |               | |                     |                    |             |                     |                         |                  | Tris          |
 The table shows compound common name, CAS number, DTXSID identifier, concentration range tested in this study, solvent used for dilutions, compound source, and the rationale for inclusion in this study. The DTXSID identifier allows linking the compound to the Computational Toxicology Dashboard (https://comptox.epa.gov/dashboard), which provides further information on structure and other chemical properties. a Mundy et al. (2015).
 . (2016).
 et al b Brown
 Abbreviation: DNTE ¼ developmental neurotoxicity evidence, DMSO ¼ dimethylsufloxide.
 |
 This timepoint was selected because it allows for sufficient time for cells to attach to the bottom of the well. Further, neurite initiation and outgrowth, as well as glial proliferation, which could impact network development, begin within the first 24hours of culture (Harrill et al. , 2011). Final concentration of DMSO, ethanol or H2O vehicle was 0.1% (vol/vol) and was without effect on network function (Brown et al. , 2016). On DIV 5 and DIV 9, all media were exchanged and compounds were readministered.
 Recordings. Spontaneous electrical activity of cortical cultures was measured using a Maestro 768-channel amplifier, Middleman data acquisition interface, and Axion Integrated Studio (AxIS) software v1.9 or later (Axion Biosystems, Atlanta, Georgia) on DIV 2, 5, 7, 9, and 12. Fifteen minutes of recording were performed immediately following 10minutes of equilibration time where the MEA plate was incubated in the Maestro at 37 /C14 C. Activity was measured using a gain of 1200 /C2 and a sampling frequency of 12.5 kHz. The signal was filtered (Butterworth band-pass filter; 0.1-5000 Hz) prior to spike detection (threshold of 8 /C2 root mean square [RMS] noise on each electrode) via the AxIS adaptive spike detector. Generally, RMS noise was between 2 and 4 m V; electrodes with RMS noise &gt; 6 m V were grounded and excluded from recordings (typically 0-4 electrodes were grounded during each recording).
 Cell viability assessment. Following the final DIV 12 MEA recording session, cell viability was assessed in the same wells by 2 assays: (1) CellTiter-Blue (CTB) Cell Viability Assay (Promega) and (2) total lactate dehydrogenase (LDH) release upon cell lysis via the CytoTox 96 Non-Radioactive Cytotoxicity Assay (Promega). Both assays were conducted exactly as described in (Brown et al. , 2017). It should be noted that the LDH assay is only measuring total cellular LDH as an indicator of how many cells are remaining in the well at DIV 12. Because of the long time between media changes and concerns regarding the half-life of LDH (approximately 9hours; see kit instructions), media concentrations of LDH were not considered reliable measures of cell death.
 Immunostaining. Cells (150 000 cells per well) were fixed on clear 48-well MEA plates on DIV 12 by replacing 300 m l of media with 200 m l of 37 /C14 C media fixation buffer (4% paraformaldehyde, 4% sucrose, and 3 m g/mL Hoechst 33342 in 1 /C2 Dulbecco's phosphate-buffered saline [DPBS]) and incubating for 20minutes at room temperature (Harrill et al. , 2011). Fixation buffer was then aspirated, and each well was rinsed 3 times with 1 /C2 DPBS. Wells were washed twice with immunostaining buffer (ISB) and then incubated with primary antibody for 2hours at room temperature. Wells were washed 3 times with ISB, incubated with secondary antibody for 1 hour at room temperature, washed twice with ISB, and washed twice with 1 /C2 DPBS. Stained cells were imaged at 10-40 /C2 with a Leica DMI6000B epi-fluorescence microscope. Primary antibodies were mouse anti-MAP2 (Millipore mAB3418, 1:400), guinea pig anti-MAP2 (Synaptic Systems 188004, 1:800), rabbit anti-GFAP (Dako Z0334, 1:1000), rabbit anti-SYP (Santa Cruz sc-9116, 1:200), rabbit anti-VGLUT1 (Synaptic Systems 135303, 1:500), mouse anti-VGAT (Synaptic Systems 131011, 1:500), and rabbit antiIBA1 (Wako 019-19741, 1:400). Secondary antibodies were goat anti-mouse AF488 (ThermoFisher A-11029), goat anti-mouse AF546 (ThermoFisher A-11030), goat anti-rabbit AF546 (ThermoFisher A-11035), goat anti-rabbit AF488 (ThermoFisher A-11034), and goat anti-guinea pig AF647 (ThermoFisher A21450), all used at 1:500.
 |
 Data analysis. All relevant data supporting this article are provided in the Supplementary Table; Supplementary Tables 1-5 are specifically referred to in the Results section, whereas Supplementary Tables 7-11 provide the raw data supporting Supplementary Tables 2-5. Axion spikelist files listing timestamped spikes were combined with comma-separated plate layout data for conversion to .h5 format files. These .h5 files were used as input to the 'meadq' (https://github.com/dianaran somhall/meadq) and 'sjemea' (https://github.com/sje30/sje mea) R packages, which together generate a set of 16 network measures per well used for all downstream analyses with date and plate ID tracking information attached. An additional network parameter, normalized mutual information (Ball et al. , in press), was computed for each recording separately. Prior to each downstream analysis, a plate-based normalization strategy was used to account for possible batch effects of different culture dates and plates. Raw values for the 17 network parameters (Supplementary Table 1) were divided by the time-matched median of each plate's 6 untreated control wells. Day in vitro 2 recordings were excluded from all quantitative analyses because it was observed that all measures of network activity were essentially zero for the majority of wells.
 To investigate the relationship between the 17 measured network parameters, a principal component analysis was performed on all data separated by DIV using the 'prcomp' function in R. Parameter relationships were further described by pairwise Pearson's correlation of the centered and scaled data. Undefined network parameter values (eg, mean interburst interval is undefined when no burst events are recorded) at DIV 5 and DIV 7 were excluded from these analyses.
 To establish potency values for compounds that impacted network activity, including both decreased terminal DIV 12 activity and delayed development of activity over time, the temporal component of each well's recordings was reduced to a single area under the curve (AUC) value, as described previously (Brown et al. , 2016). Trapezoidal AUC was calculated for each well over DIV 2-12, allowing for flexibility in developmental trajectory shape. A concentration-response relationship was then determined for each network parameter by a change in mean AUC (as percent of median untreated well AUC) over increasing concentration. If at least one of the highest 2 concentrations tested produced a change in mean AUC beyond 3 times the median absolute deviation (MAD) of all untreated control well values, a Hill function was fit to the data with a fixed upper limit of 100% and lower limit of 0%. An EC50 concentration was obtained from each curve, representing a 50% reduction or increase from untreated controls. In a few cases where the Hill curve very poorly modeled the concentration-response relationship or optimization failed, an EC50 was estimated by interpolating between the concentration that caused a greater than 50% perturbation and the next lower concentration. In all cases, if the EC50 estimate was outside the tested concentration range, it was discarded due to lack of confidence in extrapolation. Because most of these compounds have never been tested in an assay similar to the one used here, and potential effects could not be predicted, any change that resulted in an EC50 value determination for any parameter was considered a positive outcome or 'hit' in a screening context.
 Cytotoxicity (Alamar blue and LDH assay) EC50 values were determined by fitting a Hill function to DIV12 concentrationresponse plots with parameterization identical to that used for AUC calculations. Selectivity of a compound was defined as the distance from equality of network effect and cytotoxicity EC50 values (minimum of the 2 cytotoxicity assay EC50 values used).
 For selectivity and potency rankings, all compounds with no AUC EC50 values were excluded and undetermined values were set to the maximum concentration tested (30 m M). Then compounds were ordered by mean of AUC EC50 (potency) and mean selectivity across network parameters. The rule for defining the cut-off for considering an effect to be selective was based on the methods described by Stiegler et al. (2011) and Krug et al. (2013) for determining the potential specificity of neurite outgrowth inhibitors. Similar to those publications, the ratio of the EC50 for viability to the EC50 for the most sensitive endpoint for network development was evaluated for a number of tested chemicals known to be generally cytotoxic in cortical cultures ( bis tributylin oxide, colchicine, methotrexate, sodium arsenate, triethyltin and trimethylin). The mean of these ratios was 1.9 (ie, network parameters were on average affected by cytotoxic compounds at slightly lower concentrations than viability). The upper 95% confidence interval of the ratio was 2.8. Thus, a ratio of 3 was used as a threshold for compounds to be considered as having selective effects.
 Random forest classifications were performed using the R package randomForest (Liaw and Wiener, 2002) on all available data for each DIV separately. Each MEA well was labeled with either effected or normal status, where effected networks were those that were treated with compound at a concentration greater than the minimum that produced an AUC-based effect for any network parameter. A randomly selected two-thirds of the data were used for training the random forest classifier, and the withheld third was used as a test set. Undefined network parameter values were set to zero. Ten iterations were performed for each DIV, and mean accuracy was computed as the number of correct classifications over the total number of wells in the test set. Network parameter importance was determined by the mean decrease in Gini index (a measure of parameter contribution to decision tree classification accuracy), averaged across iterations, and then averaged across DIV time points for summary scores. To examine the impact of cell loss on parameter information content, the analysis was repeated with exclusion of all wells that received treatments of compound beyond the lower of the 2 cell viability EC50, as determined by Alamar blue or LDH assays.
 ## RESULTS
 ## Characterization of an MEA-Based Assay for Network Formation
 To examine the effects of a diverse set of chemicals on the in vitro development of neural networks, 86 unique compounds were tested at 7 concentrations each following the assay developed by Brown et al. (2016). Consistent with previous studies (Brown et al. , 2016; Cotterill et al. , 2016), 150 000 cells from newborn rat cortex were seeded as a dense culture on the 16-electrode field of each MEA well. Neurite outgrowth and synaptogenesis occur over the ensuing 12days to form a complex network structure. Consistent with the previous study (Brown et al. , 2016; Cotterill et al. , 2016), MEA recordings show characteristic increases in standard measures of activity and coordination over this timeframe, as illustrated by the examples in Figure 1A and B. Network activity reproducibly begins to plateau at DIV 11-13 in these cultures (Robinette et al. , 2011; Cotterill et al. , 2016), presumably concluding major network developmental events at this time. Our primary rat cortical culture has been described previously (Bjo ¨rklund et al. , 2010; Mundy and Freudenrich, 2000), but imaging of clear 48-well MEA plates confirmed cortical networks formed in this assay format contain
 |
 Figure 1. Overview of cortical network development on microelectrode arrays (MEAs). A, Representative traces of electrical activity across 4 electrodes in the same well over a 50-second time interval for 4 culture ages assayed. Note the increased complexity and coordination of activity across electrodes as days in vitro (DIV) increases. B, Boxplots showing distribution of 306 untreated control well recordings for mean firing rate (mean spikes per second [Hz] across electrodes), percent of spikes found in bursts ( /C21 5 spikes in succession), percent of spikes found in network spikes ( /C21 10 electrodes simultaneously active), and mean pairwise correlation between electrodes over 12days in culture. C, Representative images of DIV 12 cortical networks grown on 48-well MEA plates. A dense culture is maintained over the electrode array that contains microtubule-associated protein 2 (MAP2) staining of dendrites, Glial fibrillary acidic protein (GFAP)-positive astrocytes, punctate vesicular glutamate transporter 1 (VGLUT1), and vesicular GABA transporter (VGAT) staining of synaptic vesicles, punctate synaptophysin (SYP) staining of presynaptic vesicles, and a small percentage of Ionized calcium binding adapter molecule 1 (IBA1)-positive microglia. Scale bar ¼ 100 m m.
 <!-- image -->
 excitatory and inhibitory neurons, astrocytes, and a small percentage of microglia on the array at DIV 12 (Figure 1C).
 Compounds were added 2hours after cell plating and maintained in the media throughout the 12 DIV to model developmental exposure. The 86 compounds were selected for either in vivo evidence of DNT risk or to represent unknown DNT hazard. Seventeen measures of network activity and coordination were derived from each MEA well at each of the 4 time points (DIV 5, 7, 9, and 12; Supplementary Table 1). This resulted in a data set with 40 936 total endpoints measured in at least triplicate, providing an opportunity to thoroughly interrogate the connection between different network metrics and their sensitivity to chemical perturbation.
 The relationship among the 17 network parameters was first examined by a principal component analysis of all treatments. The analysis was performed separately for each time point to exclude normal developmental network changes and instead focus on chemical effects. The percent of variance explained by the first principle component was under 50% for all time points examined, and it took the top 7 principle components to explain greater than 90% of the variance in these data (Supplementary Figure 1A). Interestingly, at every time point profiled, the first principle component was aligned with a collection of 6-10 network parameters with roughly equal contribution (Supplementary Figure 1B and C). These results are consistent with our previous, more limited study (Brown et al. , 2016) and imply that the parameter space resulting from this assay is complex and not dominated by 1 or 2 network measures. In addition, the pairwise correlation (Pearson's r) between any 2 network parameters was relatively low across the 86 compound dataset (Supplementary Figure 1D). Only the number of active electrodes and number of actively bursting electrodes measures
 |
 <!-- image -->
 DIV
 Figure 2. Example data for determination of area under the curve (AUC) values for the four AUC curves in A. Note that for emamectin, values for the number of active electrodes are reduced at early days in vitro (DIV) but recover to control levels by DIV 12 at some concentrations. For all compounds, the area under each concentration across time is determined and then converted to a percentage of control wells. When 1 or more concentrations exceeded 3 times the median absolute deviation of all untreated control cultures, the data were fit to log-logistic curves to estimate the half maximal effective concentration (EC50) in a manner analogous to the EPA ToxCast project (Sipes et al. , 2013).
 shared a correlation of &gt; 0.9. These results indicate the 17 network parameters measured describe distinct aspects of neuronal network development and suggest that a consideration of multiple parameters may be helpful in assessing the full range of network changes due to compound exposure.
 ## Concentration-Dependent Effects on Network Activity
 To identify concentration-response relationships that indicated network function was impacted by compound exposure, an AUC metric was calculated for each MEA well that summed the network parameter values over the 12days. Examples of the data used to develop the AUC values are shown in Figure 2. This approach simplified the comparison of different developmental trajectories and was able to identify effects appearing as delays in the development of network activity (eg, emamectin) or as alterations in the level of one of the 17 parameters evaluated (eg, DEHP). If a compound at 1 or more concentrations produced a change in a network parameter AUC value beyond three times the MAD of all untreated control cultures, a log-logistic curve was fit to the data to estimate the half maximal effective concentration (EC50) in a manner analogous to the EPA ToxCast project (Sipes et al. , 2013). This approach was applied to all 86 compounds across the 17 network parameters (Supplementary Table 2), and the results are summarized in a heatmap (Figure 3B).
 AUC-based EC50 values were found that span the concentration range used in the assay (Figure 3A). Qualitatively, the compounds that impacted most network parameters at low concentrations were enriched for those that have evidence of in vivo DNT (Figure 3B). At least 1 network parameter was altered by 49 out of 60 DNT reference compounds (81.7%). All 4 assay positive controls reduced AUC enough to establish EC50 across multiple network parameters, whereas acetaminophen (assay negative) produced no significant perturbations. Of the unknown compounds, 11 of 21 (52.4%) had an effect on at least 1 network parameter. Most compounds exhibited effects across all network parameters at relatively similar concentrations, but a few compounds, such as 2,2',4,4'-tetrabromodiphenyl ether (PBDE-47), showed more specific patterns that impacted particular network parameters while leaving others relatively unchanged.
 ## Comparison With Multiplexed Cell Viability Assays Prioritizes Selectivity of Compounds
 In addition to the 17 network parameters quantified, 2 estimates of cell viability were made in the same culture wells after the last MEA recording on DIV12. A concentration-response curve with parameterization identical to network parameter AUC curves was fit to data from each cell viability assay to estimate an EC50 for metabolic impairment (Alamar blue) or overt cell death (LDH) (Figure 3B; Supplementary Table 3). The 2 assays provided similar results, with LDH release being slightly more sensitive, as it detected effects of 7 compounds where Alamar blue reduction did not reach the threshold for determining an EC50 value, and 5 compounds where LDH EC50 values were at least 3-fold lower than the Alamar blue values. Conversely, by the same measures, Alamar blue was more sensitive in only 2 cases (Bis 1 and thiouracil). Comparing cell viability EC50 values to the EC50 values for network effects allows for an examination of whether the effects observed on network
 |
 Figure 3. Concentration-dependent effects on network development for 86 compounds. A, Example concentration-response curves established for area under the curve (AUC) values as percent of untreated control median over increasing concentration of compound. A dotted line (orange) indicates 3 /C2 the median absolute deviation of all control wells, while a second line (gray) indicates a 50% reduction from control well median. B, Heatmap across the 86 tested compounds for concentrations that produced 50% change (EC50) from untreated control median AUC summaries of 17 measures of network activity over developmental time. Concentrations resulting in 50% reduction in 2-cell viability assays, Alamar blue conversion and lactate dehydrogenase (LDH) release, are shown in bottom rows for comparison. Color indicates log-transformed EC50 value with gray indicating no EC50 established. Color bar adjacent to compound names indicates reason for compound selection (assay positive or negative control, has in vivo developmental neurotoxicity [DNT] evidence, or represents unknown hazard). Hierarchical clustering of compounds displayed above heatmap.
 <!-- image -->
 function were specific or concomitant with cell death or loss of metabolic activity. This comparison was made for each network parameter and by using the minimum network parameter EC50 for each compound (Figure 4A). These results suggest that for all compounds that exhibited cytotoxicity, as defined by a 50% reduction in signal, there was at least 1 network effect observed at a lower concentration. However, applying a cutoff of 3-fold or greater EC50 for cell viability to EC50 for network parameter ratio (Figure 4A), some compounds caused network effects at markedly lower concentrations than that required for decreased viability, including domoic acid, haloperidol, and deltamethrin. Other compounds, like bis (tri-n-butylin) oxide, trimethyltin, and methotrexate, produced network effects at similar concentrations as those resulting in cell death or loss of metabolic activity. The relationship was further summarized by ranking compounds based on the mean potency across all network effects versus the specificity of those effects (Figure 4B; compounds circled in red are both potent and selective, those in blue are potent but less selective, and those in yellow are less potent with a range of selectivity).
 ## Machine Learning Identifies Key Network Parameters for Discriminating Chemical Effects
 Previous study in this model indicated the empirical network parameters (ie, those directly measured and always defined) were often more sensitive than parameters requiring certain bursting events to occur to be quantified (ie, derived from another measure, such as interburst interval, which requires bursting to occur). This study also demonstrated that including both overall network activity and network coordination measures are important for stratifying compound concentration-dependent effects (Brown et al. 2016). However,
 |
 <!-- image -->
 50
 Figure 4. Quantification and ranking of compound network-selective effects. A, Scatterplot comparison of the lowest network parameter EC50 value for each compound versus the lower EC50 established by cell viability assays. Values are in m M, divided by 30 m M(max tested concentration), log-transformed and negated so that a higher value indicates a more potent effect and zero corresponds to an EC50 at maximum tested concentration. The darker (black) dotted line indicates equality between cell viability and network effects, with points further above the line suggesting network effects are selective (ie, occur at concentrations lower than that required for cell loss). The lighter (blue) dotted line indicates 3-fold distance from equality as a putative cutoff for selectivity of network effects. B, Ranking of 64 compounds with network effects by least to greatest potency (mean of network parameter EC50 values) and selectivity (mean of distance from equality of each network effect EC50 vs minimumcell viability EC50). As opposed to scatterplot in A, the mean value of all 17 measures of network activity is shown to better reflect overall compound potency and selectivity. Compounds circled in red (upper right quadrant) are both potent and selective, those circled in blue are potent but less selective, and those circled in yellow (left half of figure) are less potent with a range of selectivity. Purple circles ¼ compounds with developmental neurotoxicity (DNT) evidence, pink triangles ¼ positive controls, black squares ¼ unknowns.
 these conclusions were based on testing only 5 compounds (acetaminophen was excluded because it produced no network effects) and effects on cell viability were not excluded from the analysis. Therefore, the much larger data set here provided an opportunity to revisit these questions in the context of a more diverse set of chemical perturbations and an additional network parameter (normalized mutual information). A machine-learning approach was taken to classify each MEA well in the data set as either functionally impacted or normal using random samplings of the 17 measured network parameters.
 This random forest analysis (RFA) was performed in 2 ways. In the first, individual MEA wells were labeled as 'effected' or 'normal' by including all treatments with concentrations above the lowest identified EC50 for that same compound in the effected category (Supplementary Table 4; Figure 5A). This resulted in 5931 MEA wells considered normal and 2261 labeled as functionally impacted. The second approach not only used this same stratification but also removed all wells that exhibited cytotoxicity beyond the lower cell viability EC50, leaving 1293 MEA wells labeled as functionally impacted (Supplementary Table 5; Figure 5B). In both cases, the analysis was performed for each DIV separately and one-third of the data was randomly withheld in each iteration to serve as a test set for the performance of the RFA classification.
 In both cases, receiver operating characteristic curves comparing the true- to false-positive rates demonstrate that classifier performance improved as the cultures aged from DIV 5 to DIV 12, from a mean accuracy of approximately 73%-82% (Figure 5). The poorest performing culture age was clearly DIV 5, indicating measures of network activity vary more at that time point and stabilize as the networks further develop. In both analyses, the same 4 network parameters, mean firing rate, mutual information, correlation, and burst rate, were identified as most informative for discriminating between effected and unaffected networks (Figure 5). After those 4 parameters, the order of parameter information content varies substantially between the 2 analyses, presumably due to exclusion or inclusion of MEA wells with cell loss. For example, the number of active electrodes metric is the least informative when excluding cytotoxic treatments, likely due to the low threshold (5 spikes/min) required for an electrode to be considered active. If only the top 4 parameters (mean firing rate, mutual information, correlation, and burst rate) are considered, then 52 compounds would have been identified as altering network development instead of the 64 identified using all 17 parameters (approximately 19% fewer). Of the 12 compounds that would have been missed using only the top 4 parameters, 7 (maneb, acrylamide, ketamine, naloxone, diphenylhydantoin, imidacloprid, and valproate) are from the DNT Reference Set (Mundy et al. , 2015). Taken together, these results indicate that consideration of a metric for total network activity, like mean firing rate, in combination with a measure of network synchrony, such as normalized mutual information, will identify approximately 80% of compounds that disrupt network formation. However, consideration of all 17 parameters will provide greater confidence in distinguishing chemically perturbed from normal development of cortical network function.
 ## DISCUSSION
 Together these data demonstrate the ability of an in vitro MEAbased assay to serve as a medium-throughput screening tool for detection of chemical disruptors of neural network formation. Previously, this 12-day assay was performed with a set of 6 reference chemicals for technical validation (Brown et al. 2016), but here the approach was extended to 86 unique compounds with varying evidence of in vivo DNT hazard and diverse modes of action. Of the DNT Reference compounds, 81.7% impacted
 ## Including cytotoxic treatments
 |
 <!-- image -->
 ## Summary across DIVs:
 | Parameter rank:        | 1      | 2     | 3     | 4     | 5     | 6 7   | 8           | 9     | 10    | 11    | 12    | 13    | 14    | 15    | 16 17   | |------------------------|--------|-------|-------|-------|-------|-------|-------------|-------|-------|-------|-------|-------|-------|-------|---------| | Parameter name:        | MFR MI | r     | BR    | #NS   | #SiNS | %SiNS | BD #AE      | %SiB  | NSD   | NSDsd | #ABE  | ISI   | IBI   | NSP   | NS-ISI  | | Mean Gini index: 77.02 | 70.58  | 58.29 | 52.89 | 42.65 | 37.35 | 36.77 | 36.19 35.74 | 34.56 | 33.18 | 31.70 | 31.15 | 28.85 | 27.14 | 26.51 | 23.31   |
 ## Excluding cytotoxic treatments
 <!-- image -->
 ## Summary across DIVs:
 Figure 5. Random forest classification identifies parameters best at distinguishing chemically perturbed from normal networks. A, Receive-operating characteristic curves show classification performance of random forest for treatments with network effects (concentrations beyond minimum network effect EC50) versus those without for 10 iterations. Training and performance evaluation was performed on each days in vitro (DIV) separately to focus on treatment effects. The mean decrease in Gini index (measure of information content) for each network parameter was averaged across DIV time points and a summary ranking of parameters is presented in the table below. B, Analysis in A repeated with cytotoxic treatments (treatments beyond minimum viability EC50) excluded from the random forest classifier. Note performance drops slightly across all time points, but the same 4 network parameters are found to be most informative for classification.
 | Parameter rank:   | 1     | 2 3         | 4     | 5     | 6     | 7     | 8     | 9     | 10    | 11          | 12   | 13    | 14    | 15     | 16    | 17    | |-------------------|-------|-------------|-------|-------|-------|-------|-------|-------|-------|-------------|------|-------|-------|--------|-------|-------| | Parameter name:   | MFR   | MI r        | BR    | BD    | NSD   | NSDsd | %SiB  | #SiNS | #NS   | ISI         | IBI  | %SiNS | NSP   | NS-ISI | #ABE  | #AE   | | Mean Gini index:  | 52.50 | 41.48 36.18 | 34.69 | 32.96 | 28.63 | 28.48 | 28.41 | 28.41 | 28.40 | 26.68 24.92 |      | 24.11 | 23.43 | 22.19  | 18.99 | 18.90 |
 network activity in this assay, suggesting that in vitro , most chemical DNT hazards act upon at least one process important to cortical network formation. This finding can be explained by the fact that many different fundamental neurodevelopmental processes must occur to lead to functional networks in vitro , including neurite outgrowth, maturation of glia, synaptogenesis, excitatory and inhibitory signaling, neurotransmitter recycling, and maintenance of electrochemical gradients (Johnstone et al. , 2010; Potter, 2001; Radio and Mundy, 2008).
 The ability to perform multiple recordings throughout the 12 DIV, as cortical networks develop and exhibit characteristic increases in activity and synchrony (Brown et al. , 2016; Cotterill et al. , 2016; Figure 1), is a key feature of this assay. Delays in normal network development may be just as concerning as network disruption at the final time point. Use of the AUC for summarizing network parameter values over time was favored in the analysis of these data as it captured delays in the development of network activity. Multiple compounds tested here show evidence of retarding normal network development that then recovers to normal range by DIV 12 at certain concentrations (eg, Figure 2; emamectin, see Brown et al. , 2016 for other examples). Conservatively, those compounds should also be considered potentially hazardous because the developing network likely has to adapt in some way to compensate for the actions of the compound. It is possible that delayed development would result in neural networks more susceptible to future insult due to any cellular adaptation that was required to overcome the initial exposure. Indeed, there is a body of evidence that indicates changes in the rate of neurodevelopment alter functional outcomes (Berger-Sweeney and Hohmann, 1997; Howard et al. , 2005). It will be important for future studies to challenge these recovered neural networks to examine the lasting impact of developmental exposures.
 It should be pointed out that the assay as performed does not attempt to 'wash out' compounds prior to assessment. Thus, the possibility cannot be discounted that some of the
 |
 observed effects are due to acute actions of the compounds on network activity at the time of measurement, as opposed to a developmental effect on network formation. From the standpoint of a screening approach, making such a distinction is, in our opinion, of lower priority than determining alterations in activity for several reasons. First, the data suggest that for a number of compounds, the effects following exposure during network development are different than acute effects. This includes lindane and dieldrin, which increased MFR following acute exposure (Mack et al. , 2014; Valdivia et al. , 2014) but here decreased MFR over the 12-day exposure period. For TPHP and TTDCIPP, decreases in MFR (IC50 values of approximately 16 l M) were reported by Behl et al. (2015). However, in the present study, IC50 values for MFR were not able to be determined for these compounds, and other parameters were altered at approximately 9 and 0.09 l M, respectively. These different profiles of activity indicate that not all compounds which are active in this assay are simply exerting acute effects. Second, many environmental compounds are highly lipophilic, thus, removing them by 'washing' would prove to be difficult, and thus in cases where an effect remained after washing, it still would be difficult to eliminate the possibility that acute effects were responsible for the observed changes in network function. Furthermore, in the EPA Guideline Study (OPPTS 870.6300), assessments are made on animals on PND 10, when animals are still being dosed (via the dam), and positive outcomes at this age are considered positive outcomes in the study. In this context, an acute effect is then still considered an important outcome. Finally, many compounds that are developmentally neurotoxic are also acutely neurotoxic (eg, lead, methylmercury, and chlorpyrifos oxon). Thus, from a screening standpoint, identification of compounds that alter activity during network formation, regardless of the mechanism underlying those changes, is important for initial screening approaches, such as proposed here. Compounds that simply exert an acute effect can be identified in subsequent assays.
 An important component of this assay is the inclusion of cell viability measures; they provide additional information that can be used for prioritization of compounds for further testing. By combining the recordings of network electrical activity with 2 terminal assays of cell viability, the selectivity of compound effects on network function was examined here as an example of how the data could be used in prioritization decisions. This approach facilitated ranking of compound effects on network development by both potency and selectivity (Figure 4B), an important advancement for application to DNT screening. Among the DNT Reference Set, domoic acid, haloperidol, deltamethrin, and cypermethrin were both potent and specific, whereas other compounds, including triethyltin bromide, trimethyltin, and methotrexate, were potent but nonspecific in their action. For compounds that were classified as 'unknowns' in this article, isoniazid, flusilazole, and lactofen might be considered of higher priority for additional testing than TCEP or cotinine. Although only a few examples are cited here, with a much larger number of compounds such an approach would be extremely useful for prioritization.
 Although it does make sense to prioritize substances with potent and selective effects on network development over those where changes in network function occur concomitantly with changes in viability (Smirnova et al. , 2014), nonselective effects should not be dismissed outright. All of the compounds in Figure 4 disrupted network function to some extent; thus inactive compounds are not shown, and the rankings are relative rankings among active compounds. It may be the case for the less potent and/or selective compounds that developing neurons and glia are more susceptible to decreased cell viability than are other tissue types. This hypothesis could be tested by comparison of compound effects on viability in neural versus non-neural cell types. Alternatively, compounds that are nonspecific in the MEA network formation assay may have more specific actions on other key neurodevelopmental processes, such as differentiation or proliferation. These processes may have a lesser impact on the development of networks as measured here, as the cortical cells cultured for the MEA network formation assay are largely postmitotic due to the age at which they are harvested. Examination of the specificity of these compounds on other key neurodevelopmental processes will help to clarify this question. Finally, caveats regarding the cell viability assays themselves should be considered. Because it is impractical to measure cell viability at multiple times during the assay, the viability measures were only made following the final measurement of network activity on DIV 12. Thus, cell death at an early DIV may not have been detected by changes in LDH or CTB because of compensatory responses by glia or other neurons. Thus, the lack of a response in these assays does not necessarily indicate that a compound had no adverse effects on cell health. By contrast, a decreased signal also does not necessarily indicate cell death. For example, a compound might decrease glial proliferation, resulting in lower total LDH as well as reduced metabolic activity (CTB), even though no cell death has occurred. In either case, alterations in network function would indicate the potential of the compound to alter development of the nervous system. Thus, it is important to consider the cell viability measurements as additional information in the context of screening level decision-making, not indicators of potential mechanism by which compounds may exert effects. Typically, more mechanistic information is collected in higher tier assays.
 In the case of compounds with unknown neurodevelopmental toxicity, the comparisons of potency and specificity made here allows prioritization for additional testing, whether it be in other in vitro assays, alternative species, or mammalian models. A group of 21 compounds were tested in the assay which were categorized as 'unknowns' with respect to evidence in the literature for their ability to cause DNT in mammals. The term unknown was used instead of 'negative' because the latter implies the compounds have been tested for DNT and found to have no effects. To our knowledge, this was not the case for the unknowns. These 21 compounds include a mixture of drugs (amoxicillin, captopril, fluconazole, and isoniazid), flame retardants (TDCIPP, TCEP, and TBBPA), fungicides (flusilazole, spiroxamine, and methylchloroisothiazolinone), herbicides (glyphosate and lactofen), and sweeteners (saccharin and sorbitol). It was found that 10 of the unknowns (1,2-propylene glycol, amoxicillin, chloramben, diethylene glycol, fluconazole, glyphosate, phenol, saccharin, sodium benzoate, and TBBPA) exhibited no effects on network activity over the concentration range tested in this assay (Figure 3B). Based only on the data from the network formation assay, these compounds would be of lower priority for additional screening. Some of these inactive unknowns were expected to exhibit minimal network effects. For example, sodium benzoate and propylene glycol are on the FDA Generally Regarded as Safe list (US FDA, 2015), whereas amoxicillin is considered safe for use during pregnancy (Briggs et al. , 2008). In addition, glyphosate has been shown previously to be without effects in several other assays of different neurodevelopmental processes (Williams et al. , 2000), and TBBPA was recently found to not produce a point of departure below 30 m M for acute neurotoxicity (defined as a significant drop in the
 mean firing rate of rat cortical networks with 1-hour exposure) (Behl et al. , 2015). By contrast, several of the 11 unknowns that had an effect on network function are possible DNT hazards. For instance, the tuberculosis antibiotic isoniazid has been found to exhibit acute neurotoxicity  and the organophosphorus flame retardant TDCIPP produced a point of departure in a battery of neurodevelopmental assays (Behl et al. , 2015; Jarema et al. , 2015). Interestingly, all 3 fungicide unknowns impacted network formation in this assay, with spiroxamine among the most potent compounds tested, and flusilazole relatively less potent but more selective. Based only on the present data, one might prioritize these compounds higher than TCEP, cotinine and sorbitol for additional screening.
 Measuring 17 different aspects of activity and synchrony from each cortical network provided a more complete assessment of network function than possible through any one measure. However, certain network parameters were expected to be more sensitive to chemical effects than others. The random forest classification performed here suggested the most informative network parameters for distinguishing between chemically perturbed and normal network function across time are mean firing rate, normalized mutual information, interelectrode correlation, and burst rate. This result is consistent with the generally lower concentrations required for perturbing these parameters compared with other network metrics (Figure 3A; Supplementary Table 2). Mean firing rate simply measures the total number of spikes present in the network over the MEA recording time, but exhibits a greater dynamic range than the other measures of network activity. Normalized mutual information is a measure of synchrony that scales with increasing complexity of a network. Although the low correlation coefficients and PCA results suggest the 17 network parameters are largely nonredundant and quantify different aspects of network development (Supplementary Figure 1), the machine-learning results indicate that if only a few parameters were to be quantified, a combination of a sensitive measure of network activity with a measure of network coordination would be most effective in marking network functional loss. This result is consistent with our previous, more limited examination of network parameter sensitivity (Brown et al. , 2016) and reaffirms the hypothesis that a measure of overall network activity alone (eg, mean firing rate) would be insufficient to identify major functional changes due to chemical exposures. To be clear, this RFA was designed to evaluate the utility of measuring multiple parameters of network function, not to be a prediction model for effects of any given chemical on network function. Importantly, our results also indicate that consideration of all 17 parameters will provide the most sensitivity in identification of compounds disrupting network development. Analysis of MEA data can be time and resource intensive, and in cases where such limitations exist, then evaluation of only the most sensitive parameters identified here might be justifiable. However, most vendors of MEA equipment are increasing the analysis capabilities of the data collection and analysis software provided with the equipment. Thus, going forward it will be possible to include more, rather than fewer parameters. A downside to this approach might be a higher 'false positive' rate. However, as a first-tier screen for DNT, erring on the side of more false positive responses is health protective.
 Finally, the lack of network effects of some well-known DNT compounds tested in this assay, such as phenobarbital (Motamedi and Meador, 2006; Yanai et al. , 1989) and amphetamine (Nasello and Ramirez, 1978; Vorhees, 1985), highlights the importance of considering multiple in vitro and alternative
 |
 animal model assays that cover a diversity of key neurodevelopmental processes. Eleven out of the 60 (18.3%) compounds with in vivo DNT evidence did not sufficiently perturb network function in this assay to result in a hit call. It may be the case that these compounds cause neurodevelopmental effects through mechanisms unrelated to cortical neural network development, act primarily upon stages of neurodevelopment or other neuronal cell types not present in this assay, act through metabolite production that is not present in the assay, were tested with an insufficient concentration range or produce very subtle network effects that the assay lacked sensitivity to detect. As can be expected in any chemical screening effort, effects may not be observed because concentration ranges tested did not produce an effect beyond a significant threshold. In some cases, this may be a concern because individuals are known to be exposed to very high concentrations. Indeed, this appears to be the case with valproate, where the typical test range of 0-30 m M resulted in no observed effects. However, when the valproate concentration was increased to 3 mM due to extensive literature describing neurodevelopmental effects at millimolar concentrations that can occur with therapeutic use of valproate (Roullet et al. , 2013), 3 measures of network spike behavior (the percent of spikes found in network spikes, the SD of network spike duration, and the mean number of spikes found in each network spike) were impacted at concentrations below those where substantial cell viability reduction was observed (Supplementary Table 2). Regardless of the reason for not detecting network effects for a subset of in vivo DNT hazards, these results support the view that integrating results from a battery of rapid and scalable assays will be critical for comprehensive identification of DNT hazard (Bal-Price et al. , 2010, 2015; Fritsche et al. , 2017). Monitoring multiple important neurodevelopmental processes with different sets of molecular targets will be key to complementing the shortcomings of individual assays and improving decision making that safeguards human health. Currently, many of these assays are based on human models from embryonic or inducible stem cells, whereas the MEA-based assay presented here uses rat primary cultures. In the future, it may be possible to incorporate human neural models into the network formation assay, but this will depend on the availability of human models that have an ontogeny rapid enough (2-3weeks) to make a medium throughput assay feasible. In the meantime, human models could be used in selected cases to verify results obtained in rodent cultures.
 ## CONCLUSIONS
 Our study demonstrated that an MEA-based assay for measuring development of rat cortical network activity is amenable for medium-throughput screening of environmental compounds. Application of the assay to 86 compounds revealed that most known developmental neurotoxins impact 1 or more aspects of neural activity following exposure during network formation in vitro . Additionally, many compounds exhibited selectivity by producing network activity or coordination effects at concentrations lower than those resulting in decreased cell viability. Of DNT reference compounds, 82% altered network development. Although fewer unknown compounds were tested, only 50% altered network function, and the small number of those that are generally regarded as safe were without effects, with the exception of sorbitol. Overall, these results support the hypothesis that DNT compounds will alter network activity following exposure during development at a higher rate than non-neurotoxic compounds. Going forward, this assay of neural network
 |
 function will be an important component of a battery of more rapid and scalable DNT hazard screening tools.
 ## SUPPLEMENTARY DATA
 Supplementary data are available at Toxicological Sciences online.
 ## ACKNOWLEDGMENTS
 The authors would like to acknowledge Joshua Harrill (US EPA) and Pamela Lein (UC Davis) for helpful comments on a draft of the article.
 ## REFERENCES
 - Bal-Price, A., Crofton, K. M., Leist, M., Allen, S., Arand, M., Buetler, T., Delrue, N., FitzGerald, R. E., Hartung, T., Heinonen, T., et al . (2015). International STakeholder NETwork (ISTNET): Creating a developmental neurotoxicity (DNT) testing road mapfor regulatory purposes. Arch. Toxicol . 89 , 269-287.
- Bal-Price, A. K., Hogberg, H. T., Buzanska, L., Lenas, P., van Vliet, E., and Hartung, T. (2010). In vitro developmental neurotoxicity (DNT) testing: Relevant models and endpoints. Neurotoxicology 31 , 545-554.
- Ball, K. R., Grant, C., Mundy, W. R., and Shafer, T. J. A measure of multivariate mutual information in developing neural networks in vitro is a sensitive metric for compound effects on network activity. Neural Netw . In Press. Published online 24 July 2017. https://doi.org/10.1016/j.neunet.2017.07.009.
- Behl, M., Hsieh, J.-H., Shafer, T. J., Mundy, W. R., Rice, J. R., Boyd, W. A., Freedman, J. H., Hunter, E. S., 3rd, Jarema, K. A., Padilla, S., et al . (2015). Use of alternative assays to identify and prioritize organophosphorus flame retardants for potential developmental and neurotoxicity. Neurotoxicol. Teratol . 52 , 181-193.
- Berger-Sweeney, J., and Hohmann, C. F. (1997). Behavioral consequences of abnormal cortical development: Insights into developmental disabilities. Behav. Brain Res . 86 , 121-142.
- Bjo ¨ rklund, U., Persson, M., Ro ¨ nnb € ack, L., and Hansson, E. (2010). Primary cultures from cerebral cortex and hippocampus enriched in glutamatergic and GABAergic neurons. Neurochem. Res . 35 , 1733-1742.
- Braun, J. M., Kahn, R. S., Froehlich, T., Auinger, P., and Lanphear, B. P. (2006). Exposures to environmental toxicants and attention deficit hyperactivity disorder in U.S. children. Environ. Health Perspect . 114 , 1904-1909.
- Briggs, G. G., Freeman, R. K., and Yaffee, S. J. 2008. Drugs in Pregnancy and Lactation: A Reference Guide to Fetal and Neonatal Risk . 8th ed. Lippincott Williams &amp; Wilkins, Philadelphia, PA.
- Brown, J. P., Hall, D., Frank, C. L., Wallace, K., Mundy, W. R., and Shafer, T. J. (2016). Editor's highlight: Evaluation of a microelectrode array-based assay for neural network ontogeny using training set chemicals. Toxicol. Sci . 154 , 126-139.
- Brown, J. P., Lynch, B., Curry-Chisolm, I., Shafer, T. J., and Strickland, J. D. (2017). Assaying spontaneous network activity and cellular viability using multi-well microelectrode arrays. Methods Mol. Biol. Methods Mol. Biol. 1601 , 153-170.
- Buschman, T. J., and Kastner, S. (2015). From behavior to neural dynamics: An integrated theory of attention. Neuron 88 , 127-144.
- Cotterill, E., Hall, D., Wallace, K., Mundy, W. R., Eglen, S. J., and Shafer, T. J. (2016). Characterization of early cortical neural network development in multiwell microelectrode array plates. J. Biomol. Screen 21 , 510-519.
- Fritsche, E., Crofton, K. M., Hernandez, A. F., Hougaard Bennekou, S., Leist, M., Bal-Price, A., Reaves, E., Wilks, M. F., Terron, A., Solecki, R., et al . (2017). OECD/EFSA workshop on developmental neurotoxicity (DNT): The use of non-animal test methods for regulatory purposes. ALTEX 34 , 311-315.
- Grandjean, P., and Landrigan, P. J. (2006). Developmental neurotoxicity of industrial chemicals. Lancet Lond. Engl . 368 , 2167-2178.
- Grandjean, P., and Landrigan, P. J. (2014). Neurobehavioural effects of developmental toxicity. Lancet Neurol . 13 , 330-338.
- Harrill, J. A., Freudenrich, T. M., Machacek, D. W., Stice, S. L., and Mundy, W. R. (2010). Quantitative assessment of neurite outgrowth in human embryonic stem cell-derived hN2 cells using automated high-content image analysis. Neurotoxicology 31 , 277-290.
- Harrill, J. A., Robinette, B. L., and Mundy, W. R. (2011). Use of high content image analysis to detect chemical-induced changes in synaptogenesis in vitro. Toxicol. Vitro Int. J. Publ. Assoc. BIBRA 25 , 368-387.
- Hertz-Picciotto, I., Croen, L. A., Hansen, R., Jones, C. R., van de Water, J., and Pessah, I. N. (2006). The CHARGE study: An epidemiologic investigation of genetic and environmental factors contributing to autism. Environ. Health Perspect . 114 , 1119-1125.
- Howard, A. S., Bucelli, R., Jett, D. A., Bruun, D., Yang, D., and Lein, P. J. (2005). Chlorpyrifos exerts opposing effects on axonal and dendritic growth in primary neuronal cultures. Toxicol. Appl. Pharmacol. 207 , 112-124.
- Jarema, K. A., Hunter, D. L., Shaffer, R. M., Behl, M., and Padilla, S. (2015). Acute and developmental behavioral effects of flame retardants and related chemicals in zebrafish. Neurotoxicol. Teratol . 52 , 194-209.
- Johnstone, A. F. M., Gross, G. W., Weiss, D. G., Schroeder, O. H.U., Gramowski, A., and Shafer, T. J. (2010). Microelectrode arrays: A physiologically based neurotoxicity testing platform for the 21st century. Neurotoxicology 31 , 331-350.
- Judson, R. S., Richard, A., Dix, D. J., Houck, K., Martin, M., Kavlock, R., Dellarco, V., Henry, T., Holderman, T., and Sayre, P. (2009). The toxicity data landscape for environmental chemicals. Environ. Health Perspect . 117 , 685-695.
- Karr, C. (2012). Children's environmental health in agricultural settings. J. Agromedicine 17 , 127-139.
- Korte, M., and Schmitz, D. (2016). Cellular and system biology of memory: Timing, molecules, and beyond. Physiol. Rev . 96 , 647-693.
- Krug, A. K., Kolde, R., Gaspar, J. A., Rempel, E., Balmer, N. V., Meganathan, K., Vojnits, K., Baquie ´, M., Waldmann, T., Ensenat-Waser, R., et al . (2013). Human embryonic stem cellderived test systems for developmental neurotoxicity: A transcriptomics approach. Arch. Toxicol . 87 , 123-143.
- Liaw, A., and Wiener, M. (2002). Classification and regression by randomForest. R News 2 , 18-22.
- Mack, C. M., Lin, B., Turner, J., Johnstone, A. F. M., Burgoon, L., and Shafer, T. J. (2014). Burst and principal components analysis of MEA data separates chemicals by class. Neurotoxicol . 40 , 75-85.
- Malo, N., Hanley, J. A., Cerquozzi, S., Pelletier, J., and Nadon, R. (2006). Statistical practice in high-throughput screening data analysis. Nat. Biotechnol . 24 , 167-17.
- Makris, S. L., Raffaele, K., Allen, S., Bowers, W. J., Haas, U., Alleva, E., Calamandrei, G., Sheets, L., Amcoff, P., Delrue, N., et al . (2009). A retrospective performance assessment of the developmental neurotoxicity study in support of OECD Test Guideline 426. Environ. Health Perspect . 117 , 17-25.
 - Motamedi, G. K., and Meador, K. J. (2006). Antiepileptic drugs and neurodevelopment. Curr. Neurol. Neurosci. Rep . 6 , 341-346.
- Mundy, W. R., and Freudenrich, T. M. (2000). Sensitivity of immature neurons in culture to metal-induced changes in reactive oxygen species and intracellular free calcium. Neurotoxicology 21 , 1135-1144.
- Mundy, W. R., Padilla, S., Breier, J. M., Crofton, K. M., Gilbert, M. E., Herr, D. W., Jensen, K. F., Radio, N. M., Raffaele, K. C., Schumacher, K., et al . (2015). Expanding the test set: Chemicals with potential to disrupt mammalian brain development. Neurotoxicol. Teratol . 52 , 25-35.
- Nasello, A. G., and Ramirez, O. A. (1978). Open-field and Lashley III maze behaviour of the offspring of amphetamine-treated rats. Psychopharmacology (Berl.) 58 , 171-173.
- Pola /C19 nska, K., Jurewicz, J., and Hanke, W. (2012). Exposure to environmental and lifestyle factors and attention-deficit/ hyperactivity disorder in children - a review of epidemiological studies. Int. J. Occup. Med. Environ. Health 25 , 330-355.
- Potter, S. M. (2001). Distributed processing in cultured neuronal networks. Prog. Brain Res . 130 , 49-62.
- Preziosi, P. (2007). Isoniazid: Metabolic aspects and toxicological correlates. Curr. Drug Metab . 8 , 839-851.
- Radio, N. M., and Mundy, W. R. (2008). Developmental neurotoxicity testing in vitro: Models for assessing chemical effects on neurite outgrowth. Neurotoxicology 29 , 361-376.
- Robinette, B. L., Harrill, J. A., Mundy, W. R., and Shafer, T. J. (2011). In vitro assessment of developmental neurotoxicity: Use of microelectrode arrays to measure functional changes in neuronal network ontogeny. Front. NeuroEng . 4 , 1-9.
- Roullet, F. I., Lai, J. K. Y., and Foster, J. A. (2013). In utero exposure to valproic acid and autism-a current review of clinical and animal studies. Neurotoxicol. Teratol . 36 , 47-56.
- Salinas, E., and Sejnowski, T. J. (2001). Correlated neuronal activity andtheflowofneural information. Nat. Rev. Neurosci . 2 , 539-550.
- Schmuck, M. R., Temme, T., Dach, K., de Boer, D., Barenys, M., Bendt, F., et al . (2017). Omnisphero: A high-content image analysis (HCA) approach for phenotypic developmental neurotoxicity (DNT) screenings of organoid neurosphere cultures in vitro. Arch. Toxicol . 91 , 2017-2028.
 |
 - Sipes, N. S., Martin, M. T., Kothiya, P., Reif, D. M., Judson, R. S., Richard, A. M., Houck, K. A., Dix, D. J., Kavlock, R. J., and Knudsen, T. B. (2013). Profiling 976 ToxCast chemicals across 331 enzymatic and receptor signaling assays. Chem. Res. Toxicol . 26 , 878-895.
- Smirnova, L., Hogberg, H. T., Leist, M., and Hartung, T. (2014). Developmental neurotoxicity - challenges in the 21st century and in vitro opportunities. Altex 31 , 129-156.
- Stiegler, N. V., Krug, A. K., Matt, F., and Leist, M. (2011). Assessment of chemical-induced impairment of human neurite outgrowth by multiparametric live cell imaging in high-density cultures. Toxicol. Sci. Off. J. Soc. Toxicol . 121 , 73-87.
- US FDA. GRAS Substances (SCOGS) Database. (2015). Web. 20 December 2016. https://www.fda.gov/food/ingredientspacka ginglabeling/foodadditivesingredients/ucm091048.htm. Accessed August 28, 2017.
- Valdivia, P., Martin, M., LeFew, W. R., Ross, J., Houck, K. A., and Shafer, T. J. (2014). Multi-well microelectrode array recordings detect neuroactivity of ToxCast compounds. Neurotoxicology 44 , 204-217.
- Vorhees, C. V. (1985). Behavioral effects of prenatal d-amphetamine in rats: A parallel trial to the Collaborative Behavioral Teratology Study. Neurobehav. Toxicol. Teratol . 7 , 709-716.
- Williams, G. M., Kroes, R., and Munro, I. C. (2000). Safety evaluation and risk assessment of the herbicide Roundup and its active ingredient, glyphosate, for humans. Regul. Toxicol. Pharmacol. RTP 31 , 117-165.
- Yanai, J., Fares, F., Gavish, M., Greenfeld, Z., Katz, Y., Marcovici, G., Marcovici, G., Pick, C. G., Rogel-Fuchs, Y., and Weizman, A. (1989). Neural and behavioral alterations after early exposure to phenobarbital. Neurotoxicology 10 , 543-554.
- Zimmer, B., Pallocca, G., Dreser, N., Foerster, S., Waldmann, T., Westerhout, J., Julien, S., Krause, K. H., van Thriel, C., Hengstler, J. G., et al . (2014). Profiling of drugs and environmental chemicals for functional impairment of neural crest migration in a novel stem cell-based test battery. Arch. Toxicol 88 , 1109-1126.